src/services/NeighborhoodService.js

```diff
@@ -0,0 +1,32 @@
+import * as mbxGeocoding from '@mapbox/mapbox-sdk/services/geocoding';
+import axios from 'axios';
+
+const ALL_NEIGHBORHOODS_ENDPOINT = 'https://us-central1-seattlecarnivores-edca2.cloudfunctions.net/getNeighborhoods';
+
+export default class NeighborhoodService {
+    constructor() {
+        this.geocodingService = mbxGeocoding({ accessToken: process.env.REACT_APP_MAPBOX_TOKEN});
+        console.log(process.env.REACT_APP_MAPBOX_TOKEN);
+    }
+
+    getNeighborhoodFor = (lat, lng) => {
+        // Create request
+        return this.geocodingService.reverseGeocode({
+            query: [lng, lat],
+            types: ['neighborhood']
+        })
+            .send()
+            .then(response => {
+                const neighborhoods = response.body.features;
+                if (neighborhoods && neighborhoods.length > 0) {
```

Comment:
Works perfectly! But how does the service determine which neighborhoods to look for? In other words, have you specified somewhere that we want Seattle neighborhood?

Reply:
Nope -- we still have some validation to do here for sure. I wasn't sure what the extent of our range was. What do you think?

Reply:
Oh I see! When I checked, I had not well zoomed in and was getting unknown for outside Seattle. lol! We could have a set of neighborhoods we are looking for and do a validation based on the set and the reverse geocoded value. I do not know if there is any helpful parameters in the map service that does this for us. I think we should check with Todd/Katie on the neighborhood limit.

Reply:
I think based on the terms of use for this API, we wouldn't be allowed to get a list of all neighborhoods in the Seattle area according to Mapbox and use them as a whitelist. 

If we get some bounds from Todd/Katie/an analyst then we could ensure that the point is within the bounds as a first step before even sending the data to the API.

I agree that getting some boundaries would be a good next step!